### PR TITLE
Feature/card negative

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 1]
```diff
@@ -60,6 +65,7 @@ public void iSetUpData(String data) {
         String aleatoryName = data.replaceAll("\\[(.*?)\\]", NamesGenerator.newName());
         requestManager.setData(aleatoryName);
         String completeData = Regex.getInstance().replaceID(data, context.getMapIds());
+
```
Comment (MadayAlcala) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java]: Please delete the unnecessary line.

Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java]: solved

File: src/test/resources/feature/card/card_functional_negative.feature [hunk 2]
```diff
@@ -0,0 +1,74 @@
+Feature: Functional test for "card"
+
+  @create-board @create-list @create-card @delete-card @delete-board
+  Scenario: Get a specific property that no belong of a card
+    Given I create a GET request to /cards/{idCard}/nameEndpoint endpoint
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_negative.feature]: Put a nameEndpoint more descriptive, please

Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_negative.feature]: solved

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 3]
```diff
@@ -18,11 +19,25 @@ Feature: Functional test for "card"
     When I send the request
     Then I get a 200 status code as response
 
+#  @create-board @create-list @create-card @create-action @delete-action @delete-card @delete-board
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: Delete this comments, please.

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 4]
```diff
@@ -96,23 +113,36 @@ Feature: Functional test for "card"
   @create-board @create-list @create-card @delete-card @delete-board
   Scenario Outline: Create a new checklist on a card
     Given I create a POST request to /cards/{idCard}/checklists endpoint
-      And I set up the data:
+    And I set up the data:
                   """
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: Justify this string, please

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 5]
```diff
@@ -96,23 +113,36 @@ Feature: Functional test for "card"
...
-    |name               |
-    |New name CheckList |
+      |name               |
+      |New name CheckList |
+
+  @create-board @create-list @create-card @create-label @delete-label @delete-card @delete-board
+  Scenario: Add a label to a card
+    Given I create a POST request to /cards/{idCard}/idLabels endpoint
+    And I set up the data:
+                  """
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: Justify this string, please

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 6]
```diff
@@ -96,23 +113,36 @@ Feature: Functional test for "card"
...
+                    {
+                      "value":"{idLabel}"
+                    }
+                  """
+    When I send the request
+    Then I get a 200 status code as response
 
   @create-board @create-list @create-card @create-action @delete-card @delete-board
   Scenario: Delete a comment
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: Be more specific with this scenario description, please

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 3 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: solved

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 6 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: solved

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 5 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: solved

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 4 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: solved

**Review by MadayAlcala — APPROVED** [review]
Good Job!

**Review by raulchoque-eng — APPROVED** [review]

**Pull request merged**